linux/test/debian9_nginx/Dockerfile

```diff
@@ -10,8 +10,8 @@ ARG JAVAVER=openjdk18
 ARG ICEVER=ice36
 ARG PGVER=pg96
 
-RUN echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends \
- && echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends
+RUN echo 'APT::Install-Recommends 0;' > /etc/apt/apt.conf.d/01norecommends \
+ && echo 'APT::Install-Suggests 0;' > /etc/apt/apt.conf.d/01norecommends
```

Comment:
I think this second one still needs to be an append. :smiley:

Reply:
crap too much cleaning....